### ISSUE TITLE
Fontanelle sunken

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10010]
**Preferred term label:**
Fontanelle sunken

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://www.ncbi.nlm.nih.gov/medgen/C0278207

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Marasmus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000084
- label: Sunken fontanelle
- parents: [Abnormality of fontanelles (HP:0011328)](https://hpo.jax.org/app/browse/term/HP:0011328)
- pmid: PMID:12825844
- def: Fontanelle that is more deeply set than normal. Fontanelles are normally relatively firm and curve only slightly inward. A sunken fontanelle can be a sign of dehydration or marasmus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10010
- synonyms:
